### PR TITLE
ARHZ datalogger swap

### PR DESCRIPTION
**Pull request by mnaguit**
- from Q330/3 (3616) to Centaur (9723)
- for LC 10 (EH-ENZ) streams
- Centaur firmware v4.5.20

### CODE REVIEW

**Review by salichon — DISMISSED** [review]
perfetto 
ta

**Comment by CBurton90** [comment]
@mnaguit @salichon Small change, all Centaurs will now be deployed with firmware `4.8.5` as per https://github.com/GeoNet/tickets/issues/12730#issuecomment-1530728854. Thanks.

File: install/firmware.csv [hunk 1]
```diff
@@ -458,6 +458,7 @@ Nanometrics,Centaur CTR4-6S,9711,4.5.20,2022-12-07T04:30:00Z,9999-01-01T00:00:00
 Nanometrics,Centaur CTR4-6S,9716,4.5.20,2022-12-20T22:30:00Z,9999-01-01T00:00:00Z,
 Nanometrics,Centaur CTR4-6S,9718,4.5.20,2022-12-21T22:00:00Z,9999-01-01T00:00:00Z,
 Nanometrics,Centaur CTR4-6S,9721,4.5.20,2023-03-14T23:20:00Z,9999-01-01T00:00:00Z,
+Nanometrics,Centaur CTR4-6S,9723,4.5.20,2023-05-26T03:39:00Z,9999-01-01T00:00:00Z,
```
Comment (CBurton90) [inline install/firmware.csv:461]: `Nanometrics,Centaur CTR4-6S,9723,4.8.5,2023-05-26T03:39:00Z,9999-01-01T00:00:00Z,`

**Review by CBurton90 — CHANGES_REQUESTED** [review]

**Comment by mnaguit** [comment]
FW version updated @CBurton90

**Review by CBurton90 — APPROVED** [review]

**Review by Thomas-Benson — APPROVED** [review]
Looks good to me

**Pull request merged**